docs/site.json

```diff
@@ -1,7 +1,6 @@
 {
   "baseUrl": "",
   "titlePrefix": "ConnectCS",
```

Comment:
Would it be better to change this to `ContactCS`? Since everyone else seems to be using this name when making changes, thank you!